### PR TITLE
Extentions extending

### PR DESCRIPTION
**Pull request by nikolaswise**

### CODE REVIEW

File: docs/source/assets/js/all.js [hunk 1]
```diff
@@ -2,5 +2,31 @@
...
+  window.calcite.bus.on('view:toggle', setView);
+
+  function clickHandler (e) {
+    e.preventDefault();
+    var set = e.target.getAttribute('data-toggle-set');
+    var target = e.target.getAttribute('data-toggle');
+    window.calcite.bus.emit('view:toggle', set, target);
+  }
+
+  function setView (set, target) {
```
Comment (paulcpederson) [inline docs/source/assets/js/all.js]: what does all this do?

File: docs/source/assets/js/extentions.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    e.preventDefault();
+    var set = e.target.getAttribute('data-toggle-set');
+    var target = e.target.getAttribute('data-toggle');
+    calcite.bus.emit('view:toggle', set, target);
+  }
+
+  function setView (set, target) {
+    var nodes = calcite.nodeListToArray(document.querySelectorAll('[data-toggle-set=' + set + ']'));
+    nodes.map(function (node) {
+      console.log(node)
```
Comment (paulcpederson) [inline docs/source/assets/js/extentions.js]: left some console logs in here

File: docs/source/layouts/_layout.html [hunk 3]
```diff
@@ -20,13 +20,16 @@
...
+        <link rel="stylesheet" href="{{relativePath}}assets/css/{{name}}.css">
+      {% endfor %}
     {% endblock %}
 
     <!--[if IE]>
       <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.js"></script>
       <script src="https://cdnjs.cloudflare.com/ajax/libs/selectivizr/1.0.2/selectivizr-min.js"></script>
     <![endif]-->
-
+    <script src="{{relativePath}}assets/js/libs/calcite-web.js"></script>
```
Comment (paulcpederson) [inline docs/source/layouts/_layout.html]: Assuming there was a problem that prompted this?

File: lib/sass/calcite-web/grid/_columns.scss [hunk 4]
```diff
@@ -233,6 +233,7 @@
 }
 
 @if $include-grid == true {
+  [class*="column-"] { @include column(); }
```
Comment (paulcpederson) [inline lib/sass/calcite-web/grid/_columns.scss]: nice

**Comment by paulcpederson** [comment]
a couple preguntas, but lookin sweet

**Comment by paulcpederson** [comment]
dat build, tho

File: docs/source/assets/js/all.js (hunk 1 above)
Comment (nikolaswise) [inline docs/source/assets/js/all.js]: This is some rough code that enabled interactive patterns outside of the calcite core. This is getting moved to proper extension patterns, ultimately.

File: docs/source/layouts/_layout.html (hunk 3 above)
Comment (nikolaswise) [inline docs/source/layouts/_layout.html]: yeah, but moving it back where it belongs.

File: lib/sass/calcite-web/grid/_columns.scss (hunk 4 above)
Comment (nikolaswise) [inline lib/sass/calcite-web/grid/_columns.scss]: tyvm

File: lib/js/calcite-web.js [hunk 5]
```diff
@@ -65,7 +65,7 @@ function isScrolling () {
 
 // ┌────────────────────┐
 // │ Initialize Calcite │
-// └────────────────────┘
+// └────────────────────┘q
```
Comment (paulcpederson) [inline lib/js/calcite-web.js]: ?

Comment (nikolaswise) [inline lib/js/calcite-web.js]: lol

**Comment by nikolaswise** [comment]
@paulcpederson should be good now

**Pull request merged**